matching/src/main/scala/org/kframework/backend/llvm/matching/Heuristics.scala

```diff
@@ -18,15 +18,16 @@ sealed trait Heuristic {
 
   def computeScoreForKey(c: AbstractColumn, key: Option[Pattern[Option[Occurrence]]]): Double
 
-  def getBest(cols: Seq[(Column, Int)]): Seq[(Column, Int)] = {
+  def getBest(cols: Seq[(Column, Int)], matrix: Matrix): Seq[(Column, Int)] = {
```

Comment:
let's make this method take and return a Seq[MatrixColumn] instead of a Seq[(Column, Int)].